app.js

```diff
@@ -246,6 +252,7 @@ function updateUI(dinoArray, humanData) {
   document.getElementById("repeat-btn").style.display = "block";
```

Comment:
On line 238, instead of creating a new document fragment, can you use the existing `#grid` that the index.html already provides to us?  You can see it on line 43 in app.css.

Reply:
solved in [001c905](https://github.com/leemoispace/UdacityDinosaurs/pull/1/commits/001c905e77f2367b37693a245588b8c33a47ce34)